WcaOnRails/app/webpacker/components/EditEvents/EventPanel/index.js

```diff
@@ -104,86 +104,66 @@ export default function EventPanel({
             : ''
         }
         onClick={() => dispatch(addEvent(wcifEvent.id))}
-        color="green"
-        size="tiny"
-        style={{
-          fontSize: '.75em',
-        }}
+        positive
+        size="small"
       >
         Add event
       </Button>
     );
   };
 
   return (
-    <Segment.Group
+    <Card
       size="tiny"
-      compact
       className={`event-panel event-${wcifEvent.id}`}
     >
-
-      <Header
-        className="event-panel__heading"
-        style={{
-          display: 'flex',
-          flexDirection: 'row',
-          alignItems: 'center',
-        }}
-        attached
+      <Card.Content
+        // replicate the way SemUI Cards handle images (borderless) without passing an actual image
+        style={{ padding: 0 }}
       >
-        <div
-          style={{
-            margin: '-1.5em -1em',
-          }}
-        >
-          <span className={cn('img-thumbnail', 'cubing-icon', `event-${event.id}`)} />
-        </div>
-        <span
-          className="title"
-          style={{
-            flexGrow: 1,
-            marginLeft: '2em',
-          }}
-        >
-          {event.name}
-
-        </span>
-        <div>
-          {renderRoundCountInputs()}
-        </div>
-      </Header>
-
+        <Segment basic tertiary>
+          <Header as="span">
+            <Icon className="cubing-icon" name={`event-${event.id}`} />
+            <Header.Content>
+              {event.name}
+            </Header.Content>
+          </Header>
+          <Button.Group floated="right">
+            {renderRoundCountInputs()}
+          </Button.Group>
```

Comment:
The corners are inconsistently rounded:
![image](https://github.com/thewca/worldcubeassociation.org/assets/49137025/46e9c67b-1e1a-4b4d-8698-157794f6f116)